src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -17,6 +17,14 @@ import { getGreatPrimer } from '@bbc/gel-foundations/typography';
 import { ServiceContext } from '#contexts/ServiceContext';
 import Link from './Link';
 
+const ExternalLinkTextLangs = {
+  Spotify: 'en-GB',
```

Comment:
in case someone adds another podcast service and writes something incorrect like`enGB` I'd maybe use a constant here